Userland/Libraries/LibWeb/CSS/CSSStyleDeclaration.cpp

```diff
@@ -177,7 +177,15 @@ WebIDL::ExceptionOr<String> PropertyOwningCSSStyleDeclaration::remove_property(P
     //           2. Remove that CSS declaration and let removed be true.
 
     // 6. Otherwise, if property is a case-sensitive match for a property name of a CSS declaration in the declarations, remove that CSS declaration and let removed be true.
-    removed = m_properties.remove_first_matching([&](auto& entry) { return entry.property_id == property_id; });
+    if (property_id == PropertyID::Custom) {
+        auto fly_string_property = FlyString::from_utf8(property_name);
+        if (fly_string_property.is_error()) {
+            return value;
+        }
```

Comment:
Here (and elsewhere when you do the same thing) you don't need to manually handle the error as it shouldn't happen in practice. Using `MUST()` instead is fine.